apps/passport-client/components/shared/PCDCard.tsx

```diff
@@ -24,21 +24,40 @@ export function PCDCard({
   onClick?: () => void;
   hideRemoveButton?: boolean;
 }) {
+  const [state] = useContext(DispatchContext);
   const pcdPackage = usePackage(pcd);
   const displayOptions = useMemo(() => {
     if (pcdPackage?.getDisplayOptions) {
       return pcdPackage?.getDisplayOptions(pcd);
     }
   }, [pcd, pcdPackage]);
-  const header = isZuzaluIdentity
-    ? "VERIFIED ZUZALU PASSPORT"
-    : displayOptions?.header?.toUpperCase() ?? "PCD";
+
+  const visitorStatus = getVisitorStatus(state.self);
+  const visitorExpired =
+    visitorStatus !== undefined &&
+    !visitorStatus.isDateRangeValid &&
+    visitorStatus.isVisitor;
+
+  let header = displayOptions?.header?.toUpperCase() ?? "PCD";
+
+  if (isZuzaluIdentity) {
```

Comment:
i'd consider `let expired = false` above, then

```
if (isZuzaluIdentity) {
  expired = isZuzaluIDExpired(...);
  header = expired ? "EXPIRED" : "VERIFIED ...";
}
```
that pulls a bit of zuzalu-specific logic outside of the `PCDCard` render function & ensures it only runs if `isZuzaluID`